test/controllers/licences.controller.test.js

```diff
@@ -71,4 +71,23 @@ describe('Licences controller', () => {
       })
     })
   })
+  describe('GET /licences/{id}/requirements-approved', () => {
```

Comment:
Just to break up the tests, we use a bit of whitespace to separate them.

```suggestion

  describe('GET /licences/{id}/requirements-approved', () => {
```

Reply:
✅